tests/integration/mem_leak_test.py

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
```

Comment:
The copyright year should probably be updated to 2020.

Reply:
All updated.